core/src/main/java/fr/sncf/osrd/envelope_sim/overlays/EnvelopeDeceleration.java

```diff
@@ -0,0 +1,28 @@
+package fr.sncf.osrd.envelope_sim.overlays;
+
+import fr.sncf.osrd.envelope.StepConsumer;
+import fr.sncf.osrd.envelope_sim.Action;
+import fr.sncf.osrd.envelope_sim.PhysicsPath;
+import fr.sncf.osrd.envelope_sim.PhysicsRollingStock;
+import fr.sncf.osrd.envelope_sim.TrainPhysicsIntegrator;
+
+public class EnvelopeDeceleration {
+    public static void fromTarget(
```

Comment:
Call this method `decelerate` to match the same pattern as acceleration and coast.